flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/tasks/AbstractInvokable.java

```diff
@@ -265,7 +265,7 @@ public void abortCheckpointOnBarrier(long checkpointId, Throwable cause) throws
 	 *
 	 * @return future that completes when the notification has been processed by the task.
 	 */
-	public Future<Void> notifyCheckpointCompleteAsync(long checkpointId) {
+	public Future<?> notifyCheckpointCompleteAsync(long checkpointId) {
```

Comment:
This change is not necessary. I think having a more concrete "type" (`Future<Void>`) is better here.

Reply:
removed it